include/optimizer.hpp

```diff
@@ -29,10 +30,21 @@ void Optimizer::Update(int index, NDArray weight, NDArray grad, mx_float lr) {
       param_values.data(), &handle_);
     init_ = true;
   }
-  MXOptimizerUpdate(handle_, index, weight.GetHandle(), grad.GetHandle(), lr);
+  MXOptimizerUpdate(handle_, index, weight.GetHandle(), grad.GetHandle(), learning_rate_);
```

Comment:
This usage comes from the old version c_api.h.
Please change it to the newer ones.
```
  MXNET_DLL int MXOptimizerUpdate(OptimizerHandle handle,
                                  int index,                                                                                                                 
                                  NDArrayHandle weight,
                                  NDArrayHandle grad,
                                  mx_float lr,
                                  mx_float wd); 
```